app/models/manageiq/consumption/showback_price_plan.rb

```diff
@@ -34,6 +34,14 @@ def calculate_cost(event)
     tc
   end
 
+  def bill(start_t, end_t)
+    t = Money.new(0)
+    ManageIQ::Consumption::ShowbackPool.where(:state => "CLOSED",:resource => resource).each do |pool|
+      pool.showback_charges.each do |charge|
+        t += charge.cost
```

Comment:
The idea is to get a list of charges, instead of the charges for a pool as pool.showback_charges.each